tsdb/wal/watcher_test.go

```diff
@@ -278,6 +278,8 @@ func TestReadToEndWithCheckpoint(t *testing.T) {
 					},
 				}, nil)
 				testutil.Ok(t, w.Log(series))
+				// Add in an unknown record type, which should be ignored.
```

Comment:
255 being actually a Type, to be future proof, should we e.g. write a 254 too ?

Reply:
Well not really, during type detection we only care for 1, 2, 3 etc 
![image](https://user-images.githubusercontent.com/6950331/94943513-bc5ec180-04cf-11eb-950a-a83dc4c48369.png)